server/models/Conversation.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+
+const conversationSchema = new mongoose.Schema({
+  fromUser: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'user',
+    required: true,
+  },
+  toUser: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'user',
+    required: true,
+  },
+  created: {
```

Comment:
You can use MongoDB timestamps to generate these fields automatically

Ref: https://masteringjs.io/tutorials/mongoose/timestamps